### PR TITLE
[BUG] Stripe payment dosn't work when deployed

### PR DESCRIPTION
**Pull request by offcast-dev**
PT: https://www.pivotaltracker.com/story/show/178258544
When app is deployed if trying to register 500 error appeares

## Changes Proposed
- Implements Payment Method instead of stripeToken
- refactors tests for new algorithm

### CODE REVIEW

File: config/initializers/wrap_parameters.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
+  wrap_parameters format: []
```
Comment (christerforsgren91) [inline config/initializers/wrap_parameters.rb]: why you change this?

File: app/controllers/api/registrations_controller.rb [hunk 2]
```diff
@@ -1,12 +1,20 @@
 class Api::RegistrationsController < DeviseTokenAuth::RegistrationsController
   def create
     customer = Stripe::Customer.list(email: params[:email]).data.first
-    customer ||= Stripe::Customer.create({ email: params[:email], source: params[:stripeToken] })
+    customer ||= Stripe::Customer.create({ email: params[:email]})
```
Comment (1CIM) [inline app/controllers/api/registrations_controller.rb:4]: What dose the 
`||=` really do?

File: config/initializers/wrap_parameters.rb (hunk 1 above)
Comment (offcast-dev) [reply config/initializers/wrap_parameters.rb]: We were playing with permitions in params. Also we had strange registration param we were not sending in the frontend, changed it to see what will happen and forgot to put it back))

Comment (christerforsgren91) [reply config/initializers/wrap_parameters.rb]: allright! :)

**Review by 1CIM — APPROVED** [review]

File: app/controllers/api/registrations_controller.rb (hunk 2 above)
Comment (1CIM) [reply app/controllers/api/registrations_controller.rb:4]: https://www.ruby-forum.com/t/the-definitive-list-of-or-equal-threads-and-pages/136446

Comment (1CIM) [reply app/controllers/api/registrations_controller.rb:4]: https://www.ruby-forum.com/t/the-definitive-list-of-or-equal-threads-and-pages/136446

Comment (offcast-dev) [reply app/controllers/api/registrations_controller.rb:4]: it is the same as typing unless customer in the end of the line, solargraph put pipes there. The idea is that if customer is undefined execute that line

**Review by christerforsgren91 — APPROVED** [review]

**Pull request merged**